### ISSUE TITLE
Different BF for interval Null  from  bayestestR and BayesFactor

### ISSUE THREAD

**Issue by gbiele** [easystats/bayestestR#458]
I am not sure if I am doing something wrong, but I think the following analyses should result in the same BF calculated with the `bayestestR` and `BayesFactor` packages:

```
library(rstanarm)
library(bayestestR)
library(BayesFactor)

data = data.frame(y = (sleep$extra[1:10]-sleep$extra[11:20]))
null.interval = c(-1,1)
scale = sqrt(2)/2

model <- stan_glm(
  formula = y ~ 1,
  data = data,
  prior_intercept = cauchy(0, scale, autoscale = FALSE), iter = 20000,
)

BFa = bayesfactor_parameters(model, null = null.interval)
bfInterval = ttestBF(data$y, nullInterval = null.interval, rscale = scale)
BFb = bfInterval[2] / bfInterval[1]
BFa
BFb
```
However, the results are noticeably different, such that BF_BayestesterR =  6.282 and  BF_BayesFactor =  2.108712.
[BayestesterR can also have difficulties with sampling from the prior].

The BFs are also not identical if I use a point Null:
```
BFa = bayesfactor_parameters(model, null = 0)
BFb = ttestBF(data$y, rscale = scale)
BFa
BFb
```
Here,  BF_BayestesterR =  16.058 and  BF_BayesFactor =  17.25888. This is obviously much closer.

Did I miss something that makes that the piors for the two analysis are not the same?

Any hint is very much appreciated.

I'll also post this Issue at https://github.com/richarddmorey/BayesFactor/issues.

**Comment by mattansb** [comment]
Hi @gbiele,

This isn't really an issue with `bayestestR`, but has to do with differences between `rstanarm` and `BayesFactor`. 
I will try to give the gist nonetheless (:

This all has to do with Stan (via `rstanarm`, or `brms`) using a very different parameterization than `BayesFactor`.

Just to give an example:
- In `rstanarm` the priors *are* scaled, but they are scaled by SD<sub>y</sub> = the **unconditional** SD of y.
- In `BayesFactor` the prios are also scaled, but they are scaled by σ<sub>y</sub> = the **conditional** SD of y. 

(There are many many other differences - how estimation of sigma is done, how random effects are treated... etc.)

In other words, this isn't a bug - just different flavors of priors.

Hope this helps!
